src/Conditionals.elm

```diff
@@ -0,0 +1,544 @@
+
+        9 ->
+            "September"
+
+        10 ->
+            "October"
+
+        11 ->
+            "November"
+
+        12 ->
+            "December"
+
+        _ ->
+            "Yeah, right, there's a 13th month!"
+
+
+
+-- This function trafficLightAction simulates a traffic light.
+
+
+trafficLightActionIfElse : String -> String
+trafficLightActionIfElse color =
+    if color == "Red" then
+        "Stop"
+        -- This line checks if input color is "Red" then
+        -- return "Stop"
+
+    else if color == "Yellow" then
+        "Prepare to stop"
+        -- This line checks if input color is "Yellow" then
+        -- return "Prepare to stop"
+
+    else if color == "Green" then
+        "Go"
+        -- This line checks if input color is "Green" then
+        -- return "Go"
+
+    else
+        "Are we on Mars?"
+
+
+
+-- Anything else outside of these colors ("Red", "Yellow", and "Green")
+-- then it will return "Are we on Mars?"
+
+
+trafficLightActionCase : String -> String
+trafficLightActionCase color =
+    case color of
+        "Red" ->
+            "Stop"
+
+        "Yellow" ->
+            "Prepare to stop"
+
+        "Green" ->
+            "Go"
+
+        _ ->
+            color ++ "? Are we on Mars?"
+
+
+convertTo12HourFormat : Int -> ( Int, String )
+convertTo12HourFormat hour =
+    if hour >= 0 && hour <= 23 then
+        let
+            ( convertedHour, period ) =
+                if hour < 12 then
+                    if hour == 0 then
+                        ( 12, "AM" )
+
+                    else
+                        ( hour, "AM" )
+
+                else if hour == 12 then
+                    ( 12, "PM" )
+
+                else
+                    ( hour - 12, "PM" )
+        in
+        ( convertedHour, period )
+
+    else
+        ( -1, "Extra time? This isn't football, hun!" )
+
+
+weekdayMoodIfElse : String -> String
+weekdayMoodIfElse day =
+    if day == "Monday" then
+        "Back to the grind!"
+
+    else if day == "Tuesday" || day == "Wednesday" || day == "Thursday" then
+        "Just another day in the grind"
+
+    else if day == "Friday" then
+        "TGIF!"
+
+    else if day == "Saturday" || day == "Sunday" then
+        "It's the weekend, baby!"
+
+    else
+        "Is that a public holiday?"
+
+
+weeklydayMoodCase : String -> String
+weeklydayMoodCase day =
+    case day of
+        "Monday" ->
+            "Back to the grind!"
+
+        "Tuesday" ->
+            "Just another day in the grind"
+
+        "Wednesday" ->
+            "Just another day in the grind"
+
+        "Thursday" ->
+            "Just another day in the grind"
+
+        "Friday" ->
+            "TGIF!"
+
+        "Saturday" ->
+            "It's the weekend, baby!"
+
+        "Sunday" ->
+            "It's the weekend, baby!"
+
+        _ ->
+            "Is that a public holiday?"
+
+
+fizzBuzzGame : Int -> String
+fizzBuzzGame number =
+    case ( number // 3, number // 5 ) of
```

Comment:
This is still not yet worked on.

https://github.com/chelzdev/hamelin-assignments/pull/1#discussion_r1478819197

Reply:
> This is still not yet worked on.
> 
> [#1 (comment)](https://github.com/chelzdev/hamelin-assignments/pull/1#discussion_r1478819197)

Yes but I'm still on it.